.vscode/build_vscode.py

```diff
@@ -21,8 +21,22 @@
 if args.action == "rebuild" and os.path.isdir(builddir):
     shutil.rmtree(builddir)
 
+if args.buildtype == "release":
```

Comment:
Was something broken here?
Or is there a different reason why you reverted that?

Reply:
this was to make adopting pixi optional for vscode users for now.